core/crypto/crypto.go

```diff
@@ -49,13 +49,13 @@ type Client interface {
 	Node
 
 	// NewChaincodeDeployTransaction is used to deploy chaincode.
-	NewChaincodeDeployTransaction(chaincodeDeploymentSpec *obc.ChaincodeDeploymentSpec, uuid string) (*obc.Transaction, error)
+	NewChaincodeDeployTransaction(chaincodeDeploymentSpec *obc.ChaincodeDeploymentSpec, attributes []string, uuid string) (*obc.Transaction, error)
```

Comment:
Shall we move attributes to the spec in order to preserve retro-compatibility, given also that this PR added an *attributes* field to *ChaincodeSpec*? This comment extends to all the methods that have been changed in the same sense.